docs/building_on_centos_7.md

```diff
@@ -0,0 +1,123 @@
+# Building on CentOS 7
+
+This documents step by step on how to compile and put into use the software `mod_tile` and `renderd`.
+Please see our [Continous Integration script](../.github/workflows/build-and-test-centos-7.yml) for more detail.
+
+As `CentOS 7` does not provide `mapnik`/`mapnik-devel` in the official repository (nor is it available from EPEL,) it must therefore be built and installed before `mod_tile` can be built. Although `boost` is not required (since `boost-devel` exists in the official repository,) the version available there (`1.53.0`) is not in the [recommended dependency list](https://github.com/mapnik/mapnik/blob/master/INSTALL.md#depends). There is a version of `boost` available from EPEL that might work though (`boost169-devel`.)
```

Comment:
You can replace lines @ `6-77` in this file with the contents of the version in #253.